### ISSUE TITLE
assert len(self) == len(inputs_) at \OpenNMT-py-with-BERT\onmt\modules\util_class.py", line 25, in forward

### ISSUE THREAD

**Issue by PunitShah1988** [shakeel608/OpenNMT-py-with-BERT#4]
Hi. 

I am using this module on custom data which converts english to specific pre-defined format. I created file as mentioned in read_me and did pre-processing as using preprocessing.py file. 
However, when I am trying to train it using TRANSFORMERS using below command 

train.py -data data/demo -save_model demo-model -layers 6 -rnn_size 512 -word_vec_size 512 -transformer_ff 2048 -heads 8 -encoder_type transformer -decoder_type transformer -position_encoding -train_steps 200000 -max_generator_batches 2 -dropout 0.1 -batch_size 4096 -batch_type tokens -normalization tokens  -accum_count 2 -optim adam -adam_beta2 0.998 -decay_method noam -warmup_steps 8000 -learning_rate 2 -max_grad_norm 0 -param_init 0 -param_init_glorot -label_smoothing 0.1 -valid_steps 10000 -save_checkpoint_steps 10000 -world_size 4 -gpu_ranks 0 1 2 3

I am getting error as File OpenNMT-py-with-BERT\onmt\modules\util_class.py", line 25, in forward
    assert len(self) == len(inputs_)

I am unable to understand where exactly issue lies. 

Any help is greatly appreciated. 

Thank You

**Comment by chenzelong** [comment]
I have the same mistake. Have you solved it ?
